addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -566,7 +566,7 @@
             <graph string="Workcenter Productivity" sample="1">
                 <field name="workcenter_id"/>
                 <field name="loss_id"/>
-                <field name="duration" type="measure"/>
+                <field name="duration" type="measure" string="Duration (minutes)"/>
```

Comment:
You can change the string directly in the fields data
https://github.com/odoo/odoo/blob/a52fbb1278cfeab9c53cd1f27aa98306f4974722/addons/mrp/models/mrp_workcenter.py#L326
To change it in every views